Lib9c.Tools/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Cocona;
+using Lib9c.Tools.SubCommend;
+
+namespace Lib9c.Tools
+{
+    [HasSubCommands(typeof(Create), Description = "Create commands")]
+    class Program
+    {
+        static void Main(string[] args) => CoconaLiteApp.Run<Program>(args);
+
+        public void Info() => Console.WriteLine("Show information");
```

Comment:
This line seems not fully implemented yet, in code. If it is, how about leaving *FIXME* comment?